src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -60,8 +60,6 @@ export class MeetingManager implements AudioVideoObserver {
 
   meetingId: string | null = null;
 
-  meetingRegion: string | null = null;
```

Comment:
Can you also remove it when we do cleanup and set this to null?

Reply:
I already did no?